circuitbreakerpanel/circuitbreakerpanel-java-mvn/circuitbreakerpanel-impl/src/main/resources/application.conf

```diff
@@ -0,0 +1,16 @@
+play.modules.enabled += com.lightbend.lagom.recipes.cbpanel.impl.DemoModule
+
+#For further information on configuration related to CircuitBreaker please refer to -
+# https://www.lagomframework.com/documentation/1.4.x/java/ServiceClients.html#Circuit-Breaker-Configuration
+lagom.circuit-breaker {
+
+  breakerA {
```

Comment:
The most important part of this file for the purposes of this recipe is `breakerA`. How about adding a comment highlighting it and also a reference to the sources file where `breakerA` is actually used?

I think this could help people navigate the recipe.